ibm_catalog.json

```diff
@@ -0,0 +1,73 @@
+{
+  "products": [
+    {
+      "name": "terraform-ibm-kms-key",
+      "label": "KMS Key module",
+      "product_kind": "module",
+      "tags": [
+        "dev_ops",
+        "target_terraform",
+        "terraform",
+        "module"
+      ],
+      "keywords": [
+        "terraform",
+        "key protect",
+        "hpcs",
+        "keys",
+        "encryption",
+        "kms"
+      ],
+      "short_description": "Terraform module to create a standard or root key in an IBM Key Management Service (KMS).",
+      "long_description": "Use this module to create a standard or root key in an existing key ring and IBM Key Management Services (KMS) instance. The KMS can be IBM Key Protect or IBM Cloud Hyper Protect Crypto Services(HPCS) instance. You can specify rotation and deletion policies for the keys.\n\n### Usage\n```hcl\nprovider \"ibm\" {\n  ibmcloud_api_key = \"XXXXXXXXXX\"\n  # Must be the same region the KMS instance is in\n  region           = \"us-south\"\n}\n\n# KMS root key\nmodule \"kms_root_key\" {\n  # Replace \"X.X.X\" with a release version to lock into a specific release\n  source.         = \"https://cm.globalcatalog.cloud.ibm.com/api/v1-beta/offering/source?archive=tgz&kind=terraform&name=terraform-ibm-kms-key&version=X.X.X\"\n  kms_instance_id = ibm_resource_instance.kms_instance.guid\n  key_name        = \"my-root-key\"\n}\n\n# KMS standard key\nmodule \"kms_standard_key\" {\n  # Replace \"X.X.X\" with a release version to lock into a specific release\n  source.         = \"https://cm.globalcatalog.cloud.ibm.com/api/v1-beta/offering/source?archive=tgz&kind=terraform&name=terraform-ibm-kms-key&version=X.X.X\"\n  kms_instance_id = ibm_resource_instance.kms_instance.guid\n  key_name        = \"my-standard-key\"\n  standard_key    = true\n}\n```",
```

Comment:
Can you update `kms_instance_id = ibm_resource_instance.kms_instance.guid` in both module blocks of the usage to use a mock GUID value?